src/runtime/app.ts

```diff
@@ -134,6 +165,9 @@ export class App<
 
   destroy() {
     if (this.root) {
```

Comment:
Will knowledge have a main root? If not then destroy will not work in this current state

Reply:
yes, they will have to destroy the root manually using the return object from createRoot. The code here is just for correctness.